plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -1055,6 +1056,12 @@ private static LongWriteFunction timeWriteFunction(int precision)
         String bindExpression = format("CAST(? AS time(%s))", precision);
         return new LongWriteFunction()
         {
+            @Override
+            public void setNull(PreparedStatement statement, int index) throws SQLException
+            {
+                statement.setNull(index, Types.TIME);
```

Comment:
In case it benefits other connectors as well (does it?) let's do it separately to avoid confusion when looking at history.

Reply:
> In case it benefits other connectors as well (does it?)

It may benefit _some_ connectors and it shouldn't hurt, so I am fine adding it in StandardColumnMappings.
however, _requiring_ it for everyone implementing a base-jdbc -based connector doesn't sound justified